controller/controlplane/controller.go

```diff
@@ -312,53 +193,38 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		}
 	}
 
-	log.Trace(logger, "validating ControlPlane configuration")
-	if err := validateControlPlane(cp, r.ValidateControlPlaneImage); err != nil {
-		return ctrl.Result{}, err
-	}
-
 	log.Trace(logger, "configuring ControlPlane resource")
-
 	defaultArgs := controlplane.DefaultsArgs{
 		Namespace:                   cp.Namespace,
 		ControlPlaneName:            cp.Name,
 		DataPlaneIngressServiceName: dataplaneIngressServiceName,
 		DataPlaneAdminServiceName:   dataplaneAdminServiceName,
-		AnonymousReportsEnabled:     controlplane.DeduceAnonymousReportsEnabled(r.AnonymousReportsEnabled, &cp.Spec.ControlPlaneOptions),
+		AnonymousReportsEnabled:     controlplane.DeduceAnonymousReportsEnabled(r.DevelopmentMode, &cp.Spec.ControlPlaneOptions),
```

Comment:
Why are we changing this? I believe we should stay at using the `AnonymousReportsEnabled`

Reply:
Great catch, I overlooked it during rebase